src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -1159,7 +1159,10 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 			didTapContinue: { [weak self] in
 				self?.showAntigenTestProfileInput(editMode: false)
 			},
-			dismiss: { [weak self] in self?.dismiss() }
+			dismiss: { [weak self] in
```

Comment:
I'm wondering if it wouldn't be a simpler/easier to understand later solution to do it the other way around and move the `viewModel.markScreenSeen()` in `AntigenTestProfileInformationViewController` from `viewDidLoad()` to `didTapFooterViewButton()`, then we don't need to reset it here.